parameters.cpp

```diff
@@ -437,13 +440,15 @@ bool P::addParameters() {
    RP::add("AMR.refine_on_restart","If true, re-refine vlasov grid on restart", false);
    RP::add("AMR.force_refinement","If true, refine/unrefine the vlasov grid to match the config on restart", false);
    RP::add("AMR.should_filter","If true, filter vlasov grid with boxcar filter on restart",false);
-   RP::add("AMR.refine_threshold","Determines the minimum value of the refinement parameter to refine cells", 1.0);
-   RP::add("AMR.unrefine_threshold","Determines the maximum value of the refinement parameter to unrefine cells", 0.0);
+   RP::add("AMR.use_alpha","Use alpha as a refinement index", false);
+   RP::add("AMR.alpha_refine_threshold","Determines the minimum value of alpha to refine cells", -1.0);
+   RP::add("AMR.alpha_coarsen_threshold","Determines the maximum value of alpha to unrefine cells", -1.0);
+   RP::add("AMR.use_J_per_B","Use J/B_perp as a refinement index", false);
+   RP::add("AMR.jperb_refine_threshold","Determines the minimum value of jperb to refine cells", -1.0);
+   RP::add("AMR.jperb_coarsen_threshold","Determines the maximum value of jperb to unrefine cells", -1.0);
```

Comment:
Should the defaults be sensible values instead?

Reply:
For coarsens, that defaults to half the refine threshold. I'm personally leaning more towards removing `use_whatever` and replacing that with having the refine threshold set, but either works I suppose

Reply:
Easier to switch on and off with a separate line in a cfg, but either way works.

Reply:
You could just switch it off by commenting/deleting the refine threshold since that defaults to -1

Reply:
But if we put some relatively sane thresholds as default values (which I think we should do), we need a flag for activating / deactivating the parameter.

Reply:
Defaulted to 0.5 for now